Golem/Yagna/YagnaService.cs

```diff
@@ -263,5 +263,15 @@ public Task StartInvoiceEventsLoop(CancellationToken token, IJobs jobs, EventsPu
         {
             return new InvoiceEventsLoop(Api, jobs, token, events, _logger).Start();
         }
+
+        internal IEnumerable<string> LogFiles()
+        {
+            if (!Directory.Exists(_dataDir)) {
+                return new List<string>();
+            }
+            var logFiles = Directory.GetFiles(_dataDir, "yagna_*.log");
+            var logGzFiles = Directory.GetFiles(_dataDir, "yagna_*.log.gz");
```

Comment:
Property will provide both `.log` and `.log.gz` files - otherwise I would need to unpack them somewhere.